src/HelixServer.js

```diff
@@ -63,7 +63,7 @@ function executeTemplate(ctx) {
     owner: ctx.config.contentRepo.owner,
     repo: ctx.config.contentRepo.repo,
     ref: ctx.config.contentRepo.ref,
-    path: `${ctx.resourcePath.substr(1)}.md`, // either pipeline or git-server don't like the double slash here
```

Comment:
ok, but doesn't it need to be sanitized somewhere else, then? the comment is not there for nothing,  I guess

Reply:
I assumed you put that comment and knows what is the edge case supposed to be handled there ;) Never saw 2 slashes...

Reply:
I'm regularly seeing three slashes. The problem is raw.githubusercontent.com, which does not like double-slashes. Pipeline is doing some sanitation, but I don't know if this will case cross-effects.

Reply:
Added some sanitisation that would remove any number of trailing slash.

Reply:
(don't forget to update the tests)